sdk/servicebus/service-bus/src/util/constants.ts

```diff
@@ -172,7 +172,12 @@ export const USER_METADATA = "UserMetadata";
  * @internal
  */
 export const MAX_SIZE_IN_MEGABYTES = "MaxSizeInMegabytes";
-
+/**
+ * The maximum size in megabytes.
```

Comment:
I assume you left this here to check if I was paying attention :smile:
```suggestion
 * The maximum size in kilobytes.
```